src/herbivoresAndCarnivores.js

```diff
@@ -1,16 +1,55 @@
 'use strict';
 
 class Animal {
-  // write your code here
-}
+  // static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  };
+
+  decreaseHealth(amount) {
+    this.health -= amount;
+
+    if (this.health <= 0) {
+      this.die();
+    }
+  };
+
+  die() {
+    const index = Animal.alive.indexOf(this);
+
+    if (index !== -1) {
+      Animal.alive.splice(index, 1);
+    }
```

Comment:
Don't manipulate with array in this way, use filter method instead

Reply:
Why?

Reply:
because this is a more obvious method